translate/cloud-client/translate_v3_translate_text_with_model_test.py

```diff
@@ -0,0 +1,28 @@
+# -*- encoding: utf-8 -*-
```

Comment:
Delete this line

Reply:
I think since there is some japan letters, i can't delete that. Is there other way to include non-ascii values?
` SyntaxError: Non-ASCII character '\xe3' in file /tmpfs/src/github/python-docs-samples/translate/cloud-client/translate_v3_translate_text_with_model_test.py on line 27`